README.md

```diff
@@ -126,33 +134,16 @@ scripts/helm/delete
 
 ### Accessing the pod
 
-The ffc-demo-user-service is not exposed via an endpoint within Kubernetes.
-
-The deployment may be accessed by forwarding a port from a pod.
-First find the name of the pod by querying the namespace, i.e.
-
-`kubectl get pods --namespace ffc-demo-user-service-pr2`
-
-This will list the full name of all the pods in the namespace. Forward the pods exposed port 3002
-to a local port using the name returned from the previous command, i.e.
+By default, the service is not exposed via an endpoint within Kubernetes.
 
-`kubectl port-forward --namespace ffc-demo-user-service-pr2 ffc-demo-user-service-8b666f545-g477t  3002:3002`
-
-Once the port is forwarded a tool such as [Postman](https://www.getpostman.com/) can be used to access the API at http://localhost:3002/register.
-
-Sample valid JSON that can be posted is:
+Access may be granted by forwarding a local port to the deployed pod:
 
 ```
-{
-  "email": "test@email.com"
-}
+# Forward local port to the Kubernetes deployment
+kubectl port-forward deployment/ffc-demo-user-service 3002:3002

```

Comment:
it's best to explicitly include the namespace in the sample command for clarity, i.e. 
`kubectl port-forward -n default deployment/ffc-demo-user-service 3002:3002

Reply:
Good catch. I've added the `ffc-demo-user-service` namespace since it's specified in the helm install script.